### PR TITLE
Add chases2 to test-infra oncall

### PR DESCRIPTION
**Pull request by chases2**
/assign @fejta 
/cc @cjwagner

### CODE REVIEW

File: config/kubernetes/sig-testing/teams.yaml [hunk 1]
```diff
@@ -59,6 +59,7 @@ teams:
     members:
     - amwat
```
Comment (fejta) [inline config/kubernetes/sig-testing/teams.yaml]: Can you drop amwat, ixdy, krzyzacy?

Comment (chases2) [reply config/kubernetes/sig-testing/teams.yaml]: Should I drop them from `sig-testing-pr-reviews` and/or `test-infra-maintainers` as well?

Comment (fejta) [reply config/kubernetes/sig-testing/teams.yaml]: Let's leave those alone

Comment (chases2) [reply config/kubernetes/sig-testing/teams.yaml]: Changes have been made

**Pull request merged**